contracts/handler/Aave/AaveAssetHandler.sol

```diff
@@ -568,16 +575,24 @@ contract AaveAssetHandler is IAssetHandler {
       DATA_PROVIDER_ADDRESS
     ).getUserReserveData(_underlyingToken, _user);
 
+    console.log("_underlyingToken in contract", _underlyingToken);
+    console.log("currentVariableDebt in contract", currentVariableDebt);
+
     //Convert underlyingToken to 18 decimal
     uint borrowBalance = currentVariableDebt *
       10 ** (18 - IERC20MetadataUpgradeable(_underlyingToken).decimals());
 
+    console.log("borrowBalance in contract", borrowBalance);
+
     //Get price for _protocolToken token
     uint _oraclePrice = IAavePriceOracle(PRICE_ORACLE_ADDRESS).getAssetPrice(
       _underlyingToken
     );
+
+    console.log("_oraclePrice in contract", _oraclePrice);
     //Get price for borrow Balance (amount * price)
-    uint _tokenPrice = (borrowBalance * _oraclePrice) / 10 ** 8;
+    uint _tokenPrice = (borrowBalance * _oraclePrice) / 10 ** 18;
```

Comment:
Does this only support tokens with 18 decimals?

Reply:
It supports all type of tokens, but converts to 18 decimal

Reply:
But thanks for pointing out, I guess there is an issue here, not all aave lending tokens are 18 decimal

Reply:
It's correct checked it and our collateral amount to sell is correct

Reply:
For any kind of token?